apps/mobile/src/components/widgets/accounts/accounts-widget.tsx

```diff
@@ -66,8 +63,7 @@ export function AccountsWidget({ accounts, wallets, totalBalance }: AccountsWidg
               router.navigate({
                 pathname: AppRoutes.Account,
                 params: {
-                  fingerprint: account.fingerprint,
-                  account: account.accountIndex,
+                  accountId: account.id,
```

Comment:
@edgarkhanzadian : I changed this code to pass the `accountId` and then use `destructAccountIdentifer` to get the `fingerprint` + `accountIndex` on arrival at `AppRoutes.Account`. 

Should I refactor `[account-selector-sheet.tsx](https://github.com/leather-io/mono/pull/407/files#diff-c51bb2ab3f10a009531e45d8e5b7fd372b2e54434dafa8925304560c03352c63)` to do that also? 

It destructs the identifier then passes them as separate params so I'm not sure whats best
```
 const onAccountPress = useCallback(
    (accountId: string) => {
      const { fingerprint, accountIndex } = destructAccountIdentifer(accountId);
      modalRef.current?.close();
      router.navigate({
        pathname: APP_ROUTES.WalletWalletsSettingsConfigureAccount,
        params: { wallet: fingerprint, account: accountIndex },
      });
    },
    [router, modalRef]
  );
```

Reply:
OK so maybe I should revert my other code to keep passing `fingerprint` + `accountIndex`? And ignore `destructAccountIdentifer`